arango/collection.py

```diff
@@ -335,6 +337,26 @@ def response_handler(resp: Response) -> Json:
 
         return self._execute(request, response_handler)
 
+    def info(self) -> Result[Json]:
+        """Return the collection information.
```

Comment:
not sure if this method should live in `collection.py` or `database.py`, given the precedent of storing certain  `/_api/collection/{name}` endpoints in `database.py` ([source](https://github.com/ArangoDB-Community/python-arango/blob/main/arango/database.py#L901-L1145))

Currently, UX would look like

```py
db.collection('myCollection').info()
```

Alternatively we could have something like:
```py
db.collection_info('myCollection')
```

I have no preference

Reply:
I believe the current implementation is more user-friendly. It lets users pass around the collection object without worrying about its name or the database.
```python
def foo(col: Collection):
    col.info()
```
Regarding `view_info()`, unfortunately there's no `View` class.

Reply:
makes sense, will keep it as is 👍